src/app/gulpfile.js

```diff
@@ -14,6 +14,13 @@ const collectionEditor = process.env.sunbird_collection_editor_artifact_url;
 const genericEditor = process.env.sunbird_generic_editor_artifact_url;
 const editorsDestPath = 'client/src/thirdparty/editors/'
 
+// PhraseApp Configuration
+const authToken = 'Basic YXJqdW5raGVjeaE6QXJqdW5AMTk5MA==';
+const project = 'DIKSHA Portal,Sunbird Creation ';
```

Comment:
Tenant references should not be there

Reply:
There is No Tenant references It is Phraseapp Project Name.